src/api/app/components/request_decision_component.html.haml

```diff
@@ -1,33 +1,60 @@
+              .d-flex.justify-content-end
+                = render CannedResponsesDropdownComponent.new(User.session.canned_responses.where(decision_type: nil).order(:title))
+            = text_area_tag(:reason, nil,
+                            placeholder: 'Write your comment or decision...(markdown is only supported for comments, not for decisions)',
+                            rows: 4, class: 'w-100 form-control mb-2 message-field')
+          .tab-pane.fade{ id: 'preview_new_comment', role: 'tabpanel', 'aria-labelledby': 'preview-message-tab' }
+            .comment-preview.message-preview.border-bottom.border-gray-300.my-3
+      - if single_action_request && @is_target_maintainer && @bs_request.state.in?([:new, :review])
+        - if show_add_submitter_as_maintainer_option?
+          .form-check.mb-2.mt-2
+            = check_box_tag('add_submitter_as_maintainer_0', "#{@action.target_project}_#_#{@action.target_package}",
+                            false,
+                            class: 'form-check-input')
+            %label.form-check-label{ for: 'add_submitter_as_maintainer_0' }
+              Add #{link_to(@bs_request.creator, user_path(@bs_request.creator))} as maintainer for
+              #{helpers.project_or_package_link(project: @action.target_project, package: @action.target_package, short: true)}
+        - if @action.type == 'submit'
+          - @action.forward.each_with_index do |forward, index|
+            .form-check.mb-2
+              = check_box_tag("forward-#{index}", "#{forward[:project]}_#_#{forward[:package]}", true, class: 'form-check-input')
+              %label.form-check-label{ for: "forward-#{index}" }
+                Forward submit request to
+                #{helpers.project_or_package_link(project: forward[:project], package: forward[:package], short: true)}
+      - if policy(@bs_request).revoke_request?
+        = submit_tag 'Revoke request', name: 'revoked', class: 'btn btn-danger me-2', data: other_decision_confirmation('revoke')
+      - if policy(@bs_request).reopen_request?
+        = submit_tag 'Reopen request', name: 'new', class: 'btn btn-warning me-2', data: other_decision_confirmation('reopen')
+      - if policy(@bs_request).accept_request?
+        - if policy(@bs_request).decline_request?
+          = submit_tag('Decline request', name: 'declined', title: 'Decline the request, as the author of the request you can reopen it again.',
+                      class: 'btn btn-danger me-2', data: other_decision_confirmation('decline'))
+        = submit_tag('Accept request', name: 'accepted', title: 'Accept the request, this will apply the changes on the target.',
+                    class: 'btn btn-secondary me-2', data: confirmation)
+      - if policy(Comment.new(commentable:@bs_request)).create?
+        = submit_tag 'Add comment', class: 'btn btn-primary me-2', data: { disable_with: 'Creating comment...' }, name: 'commented'
```

Comment:
I think this should be `btn-secondary`

Reply:
Also switching `Accept` to `btn-primary`, do you agree?

Reply:
ah, just saw the other comment :+1: